mocks/mocks.py

```diff
@@ -723,6 +743,40 @@ def project_3D_to_2D(particle_positions,
     return particle_positions_2D, x * u.kpc, y * u.kpc
 
 
+def create_cosmology(costum_cosmology_parameters,
+                     default_cosmology=WMAP9):
+    """
+    Customizes the cosmology parameters.
+
+    Parameters
+    ----------
+    costum_cosmology_parameters : dict
+        Contains preferred cosmology parameters.
+    default_cosmology : TYPE, optional
+        The main cosmology, default WMAP9.
+
```

Comment:
I think this docstring should explain a bit more how the `custom_cosmology_parameters` and `default_cosmology` are used. It's clear enough from reading the code, but if I only had the docstring it wouldn't be obvious to me that this function creates a flat Lambda CDM cosmology using the `hubble_constant`, `Omega_0`, `T_CMB0`, `N_eff`, `mass_nu` and `Omega_b0` parameters from `custom_cosmology_parameters` or, for parameters that aren't in `custom_cosmology_parameters`, from `default_cosmology`.